Rlabkey/NEWS

```diff
@@ -1,5 +1,5 @@
-Changes in version ???
-  o Fix typo in labkey.webdav.put for generated Rlabkey.pdf (Issue 37739)
```

Comment:
Looks like your change for the note about issue 38427 remove the line that existed in this NEWS doc for 37739. Can you add that back in?

Reply:
Yup - done!